src/containers/AppBody.jsx

```diff
@@ -15,31 +15,38 @@ const API_URL = process.env.REACT_APP_API_URL;
 
 export class AppBody extends React.Component {
   componentDidMount() {
-      const { username } = this.props.params;
-      const { id } = this.props.params;
-      const { dispatch } = this.props;
-      if (!username && !id) {
-        const signInState = cookie.load('signInState');
-        if (signInState) {
-          cookie.remove('signInState', { path: '/' });
-          cookie.remove('signInRedirect', { path: '/' });
-          dispatch(restoreState(signInState));
-        }
-        return null;
+    const {
```

Comment:
Could you please add a comment here, such as `// Extract variables from this.props`? This threw me off initially, as it doesn't quite look like typical destructuring syntax (to me, at least).